dnf-behave-tests/dnf/builddep.feature

```diff
@@ -32,6 +33,8 @@ Scenario: Builddep with simple dependency (srpm)
         | Action        | Package                           |
         | install       | lame-libs-0:3.100-4.fc29.x86_64   |
 
+# @dnf5
+# TODO(nsella) rpmdb check fail
```

Comment:
This one should pass now - PR https://github.com/rpm-software-management/libdnf/pull/1502 fixes this issue.